builtin/locale/__builtin.fr.tr

```diff
@@ -0,0 +1,246 @@
+<privilege>=<privilège>
+Return list of all online players with privilege=Renvoie la liste de tous les joueurs en ligne avec un privilège.
+Invalid parameters (see /help haspriv).=Paramètres invalides (voir /help haspriv)
+Unknown privilege!=Privilège inconnu !
+No online player has the "@1" privilege.=Aucun joueur en ligne avant le privilège « @1 »
+Players online with the "@1" privilege: @2=Joueurs en ligne avec le privilège « @1 » : @2
+Your privileges are insufficient.=Vos privilèges sont insuffisants.
+Your privileges are insufficient. '@1' only allows you to grant: @2=Vos privilèges sont insuffisants. '@1' vous autorise seulement d'accorder : @2
+Unknown privilege: @1=Privilège inconnu : @1
+@1 granted you privileges: @2=@1 vous a accordé les privilèges : @2
+<name> (<privilege> [, <privilege2> [<...>]] | all)=<nom> (<privilège> [, <privilège2> [<...>]] | all)
+Give privileges to player=Accorder le privilège au joueur
+Invalid parameters (see /help grant).=Paramètres invalides (voir /help grant)
+<privilege> [, <privilege2> [<...>]] | all=<privilège> [, <privilège2> [<...>]] | all
+Grant privileges to yourself=Accorder des privilèges à vous-même
+Invalid parameters (see /help grantme).=Paramètres invalides (voir /help grantme)
+Your privileges are insufficient. '@1' only allows you to revoke: @2=Vos privilèges sont insuffisants. '@1' vous autorise seulement à révoquer : @2
+Note: Cannot revoke in singleplayer: @1=Note : Impossible de révoquer en solo : @1
+Note: Cannot revoke from admin: @1=Note : Impossible de révoquer à l'administrateur : @1
+No privileges were revoked.=Aucun privilège n'a été révoqué.
+@1 revoked privileges from you: @2=@1 vous a révoqué le privilège : @2
+Remove privileges from player=Révoquer les privilèges au joueur
+Invalid parameters (see /help revoke).=Paramètres invalides (voir /help revoke).
+Revoke privileges from yourself=Révoquer des privilèges à vous-même
+Invalid parameters (see /help revokeme).=Paramètres invalides (voir /help revokeme).
+<name> <password>=<nom> <mot de passe>
+Set player's password (sent unencrypted, thus insecure)=Voir le mot de passe d'un joueur (envoyé non crypté, soit non sécurisé)
+Name field required.=Le champ « nom » est requis.
+Your password was cleared by @1.=Votre mot de passe a été effacé par @1.
+Password of player "@1" cleared.=Mot de passe du joueur « @1 » effacé.
+Your password was set by @1.=Votre mot de passe a été défini par @1.
+Password of player "@1" set.=Mot de passe « @1 » défini.
+<name>=<nom>
+Set empty password for a player=Définir un mot de passe pour un joueur
+Reload authentication data=Recharger les données d'authentification
+Done.=Fait.
+Failed.=Échoué.
+Remove a player's data=Supprimer les données d'un joueur
+Player "@1" removed.=Joueur « @1 » supprimé.
+No such player "@1" to remove.=Aucun joueur « @1 » à supprimer.
+Player "@1" is connected, cannot remove.=Le joueur « @1 » est connecté, impossible de supprimer.
+Unhandled remove_player return code @1.=
```

Comment:
This has not been translated.

Reply:
Yes I did not understood if I should interpret as `remove_player` or `remove player`. Because I haven't seen it elsewhere in the file.

Reply:
It is the name of a chat command (not translated):
https://github.com/minetest/minetest/blob/8ef2c42150123c4a626925f554dd74700c8b6dfe/builtin/game/chat.lua#L484-L507